src/cursor.c

```diff
@@ -250,16 +198,26 @@ cursor_set(struct seat *seat, const char *cursor_name)
 	}
 
 	/* Prevent setting the same cursor image twice */
-	if (last_cursor_image) {
-		if (!strcmp(last_cursor_image, cursor_name)) {
-			return;
-		}
-		free(last_cursor_image);
+	if (!g_strcmp0(seat->server_cursor_name, cursor_name)) {
```

Comment:
What is your reasoning for `g_strcmp0` vs `strcmp`?

Reply:
Probably to avoid having to check for NULL pointers

Reply:
Hm.. Me personally would prefer to keep it base C where ever possible. I know we are already using glib in other parts for splitting strings and similar but in this case I don't really see an advantage vs using an additional `if (seat->server_cursor_name)`.

Reply:
`g_strcmp0` is more readable in my opinion, but I have no problem spelling out the `NULL` check if that's preferred.
`g_strcmp0` is already used in `is_title_different()` (osd.c) as well, for what it's worth.

Reply:
Well, its a project decision /me thinks. As such: thoughts @johanmalm?
Just because I prefer one variant doesn't mean that everybody else has to agree. I am fine with whatever the project lead decides.